lib/puma/cluster.rb

```diff
@@ -434,7 +434,7 @@ def run
                 case req
                 when "b"
                   w.boot!
-                  log "- Worker #{w.index} (PID: #{pid}) booted, phase: #{w.phase}"
+                  log "- Worker #{w.index} (PID: #{pid}) booted in #{w.uptime.truncate(*(RUBY_VERSION >= "2.4." ? [2] : []))}s, phase: #{w.phase}"
```

Comment:
```suggestion
                  log "- Worker #{w.index} (PID: #{pid}) booted in #{w.uptime.round(2)}s, phase: #{w.phase}"
```

Should also get us decimals when using Ruby < 2.4